### DISCUSSION TITLE
Cannot add EAN for product

### DISCUSSION THREAD

**Discussion by minhcz** [PrestaShop/PrestaShop#31365] [Q&A] [answered]
Hello,

During insert product to Prestashop i got problem is cannot add EAN for product. After insert and save successful, reopen product editor that EAN field empty. Anyone faced this bug?

Version 8.0.1 Multistore, php 8.1.2, cache off.

Thank you

**Comment by FabienPapet** [comment, marked as answer]
Hello,

If you think this is a bug inside Prestashop, please create an issue instead of a discussion.

Moving this to https://github.com/PrestaShop/PrestaShop/issues/31396
